frontend/TestRun/TestRun.svelte

```diff
@@ -275,6 +278,16 @@
                         role="tab"
                         ><Fa icon={faExclamationTriangle}/> Activity</button
                     >
+                    <button
```

Comment:
Maybe put it somewhere in the middle, for example next to the potential jUnit tab.

Reply:
moved